test/benchmark_http_client_test.cc

```diff
@@ -26,6 +26,37 @@ using namespace testing;
 
 namespace Nighthawk {
 
+namespace {
+const std::shared_ptr<Envoy::Http::RequestHeaderMap>& getDefaultHeaderMap() {
```

Comment:
Even though optional, can we add at least a single line of comment explaining what this function does?

Reply:
Removing this function and replacing with a member.